platform-validation/scripts/compatibility.sh

```diff
@@ -490,14 +531,31 @@ check_systemd(){
     fi
 }
 
+cat /etc/os-release
 get_architecture
 echo "Architecture:$ARCH"
 echo "OS Type:$OSTYPE"
 
-#TODO : Do we need to check in both host and container?
-check_net_cap_bind_host
-check_net_cap_bind_container
 check_cgroup_heirachy
+#Flags Required for setting elevated capabilities in a container.
+check_kernel_flags EXT4_FS_SECURITY
```

Comment:
we could let Cx know if the elevation is not met as a warning and launch with right priveleges?

Reply:
No, The Customer would need to rebuild the kernel.